index.js

```diff
@@ -580,6 +593,150 @@ module.exports = {
 
     self.rewriteUrl = url => {
       return url;
-    }
+    };
+
+    self.getPagesTree = async (req) => {
+      self.maps = {};
+      const excludedTypes = [
+        'workflow-document',
```

Comment:
I don't understand why bu I was getting twice each piece because of this module type, I didn't find it doesn't happen in the normal `self.getPieces` method, so I did this but I suppose there is a better way.

Reply:
This actually does make sense. The problem is that `workflow-document` is a read-only "virtual piece type" that intentionally includes all doc types in order to implement the "manage workflow" view. Good catch.